src/ansible/variables.yml

```diff
@@ -45,7 +45,7 @@ user: {
     password: Secret123
   },
   regular: {
-    uid: 1000,
+    uid: 1024,
```

Comment:
This is not needed. The variable should be overwritten when running the playbook from te metadata.

Reply:
It does need to be overridden this way so it is clearly better this way.